### DISCUSSION TITLE
Github Repository Deployment Enviornment "Environment variables"

### DISCUSSION THREAD

**Discussion by clemenstan** [integrations/terraform-provider-github#1549] [Ideas]
Hello, 
Github just release this year the option to also create "environment variables" instead of only "environment secrets" in for github deployment environments. 
Could you please add this feature to the Github-Provider?
Thanks in prior!

Best 
Clemens

**Comment by laughedelic** [comment]
This was added in
- https://github.com/integrations/terraform-provider-github/pull/1569
- https://registry.terraform.io/providers/integrations/github/latest/docs/resources/actions_environment_variable
